stories/components/Dropdown.stories.tsx

```diff
@@ -6,6 +6,9 @@ import Button from '../../src/components/deprecated-button'
 export default {
     title: 'Components/Dropdown',
     component: Dropdown,
+    parameters: {
+        badges: ['notAccessible'],
```

Comment:
I would deprecate this one as well. I'm pretty certain that we're going to introduce a different component for this purpose.